hbase-backup/src/main/java/org/apache/hadoop/hbase/backup/BackupDriver.java

```diff
@@ -187,10 +188,7 @@ public static void main(String[] args) throws Exception {
 
   @Override
   public int run(String[] args) throws IOException {
-    if (conf == null) {
-      LOG.error("Tool configuration is not initialized");
-      throw new NullPointerException("conf");
-    }
+    Objects.requireNonNull(conf, "Tool configuration is not initialized");
```

Comment:
Looks nice. Is there an advantage to this approach? Thanks.

Reply:
@saintstack Less code.  I also like the JavaDoc for the method.  I copy & paste its message to use the same helpful formatting. 

Trying to introduce it so that others may stumble upon and learn.  Best used in this way:

```
public MyClass(String val) {
  this.val = Objects.requireNonNull(val);
}
```

More concise than the alternative:

```
public MyClass(String val) {
  if (val == null) {
    throw new NullPointerException();
  }
  this.val = val;
}
```

But if it's going to be used one way, might as well be used everywhere.  Pretty versatile that way.